ACKNOWLEDGEMENTS.md

```diff
@@ -1,5 +1,4 @@
-Acknowlegements
-===============
+# Acknowlegements #
```

Comment:
Is the change from `===` to `##` mandatory for the lint to pass? Can it be excepted? The reason the underlining is used is that the .md file is supposed to be well readable in the pure textual form.